frontend/src/components/PartnerList.vue

```diff
@@ -175,35 +177,38 @@ export default {
       });
     },
 
-    fetch() {
-      this.$store.dispatch("loadPartnerSelectList");
-
-      partnerService
-        .getPartners(this.onlyIssuersForSchema)
-        .then((result) => {
-          console.log("Partner List", result);
-          if (Object.prototype.hasOwnProperty.call(result, "data")) {
-            this.isBusy = false;
-
-            if (this.onlyAries) {
-              result.data = result.data.filter((item) => {
-                return item.ariesSupport === true;
-              });
-            }
-
-            this.data = result.data.map((partner: PartnerAPI) => {
-              const tempPartner: PartnerAPI & { address: string } = {
-                address: this.getProfileAddress(partner),
-                ...partner,
-              };
-              return tempPartner;
+    async fetch() {
+      this.isBusy = true;
+      // this.$store.dispatch("loadPartnerSelectList");
+      this.items = [];
+      const params = PageOptions.toUrlSearchParams(this.options);
+      try {
+        const response = await partnerService.getPartners(
+          this.onlyIssuersForSchema,
+          this.showInvitations,
+          params
+        );
+        if (response.status === 200) {
+          const { itemsPerPage } = this.options;
+          if (this.onlyAries) {
```

Comment:
local filtering on the page breaks the pageable I guess.

Reply:
This property comes from a time where we had the first version of the web-only mode, there it was possible to start the BPA without aca-py and you could connect via public profile only. So it made sense to filter partners with no aries capability from lists where it was required, like all exchanges. The current web-only mode works different by activating the ledger read only flag, so there is always an aries endpoint to talk to. I see that this property is only used in this component so it and the filter can be safely removed I guess. If the partner has no aries endpoint the connection will get stuck in state request-sent anyway. For better user experience we could throw an exception in the PartnerLookup and then remove this property altogether, but I would do this in a second step, right know I would only remove it from this component and do the backend later.